.circleci/config.yml

```diff
@@ -1,24 +1,79 @@
-version: 2
+version: 2.1
 jobs:
-  build:
+  deps:
     docker:
       - image: circleci/node:10.15
     steps:
       - checkout
       - restore_cache:
           keys:
-          - v1-dependencies-{{ checksum "package.json" }}-{{ checksum "yarn.lock" }}
-          - v1-dependencies-{{ checksum "package.json" }}
-          - v1-dependencies-
+            - v1-dependencies-{{ checksum "package.json" }}-{{ checksum "yarn.lock" }}
+            - v1-dependencies-{{ checksum "package.json" }}
+            - v1-dependencies-
       - run: yarn install
       - save_cache:
           paths:
+            - .git
             - node_modules
           key: v1-dependencies-{{ checksum "package.json" }}-{{ checksum "yarn.lock" }}
-      - run: yarn license-check
-      - run: yarn test-alex
+
+  lint:
+    docker:
+      - image: circleci/node:10.15
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v1-dependencies-{{ checksum "package.json" }}-{{ checksum "yarn.lock" }}
       - run: yarn lint
-      - run: yarn flow:ci
+
+  tests:
+    docker:
+      - image: circleci/node:10.15
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v1-dependencies-{{ checksum "package.json" }}-{{ checksum "yarn.lock" }}
       - run: yarn test --coverage --runInBand
-      - run: yarn build-prod:quiet
       - run: bash <(curl -s https://codecov.io/bash)
+
+  build-prod:
+    docker:
+      - image: circleci/node:10.15
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v1-dependencies-{{ checksum "package.json" }}-{{ checksum "yarn.lock" }}
+      - run: yarn build-prod:quiet
+
+  others:
```

Comment:
I would call this `flow-license-alex`. It might be better if we could even do something more explicit like `Flow License and Alex Checks`

<img width="681" alt="image" src="https://user-images.githubusercontent.com/1588648/66409716-82d7a100-e9b6-11e9-8f90-4226811fd75d.png">
That way this is more descriptive

Reply:
Actually I think we can add a description, I'll look into it.